css/style.css

```diff
@@ -24,59 +36,55 @@ a {
     text-decoration: underline;
 }
 
-
 a:hover {
     text-decoration: none;
 }
 
-#main {
-    width: 912px;
-    margin: 0 auto;
-    padding: 0;
+img {
+    max-width: 100%
 }
 
-#nav {
-    float: right;
-    color: #FFF;
-    margin: 10px 18px 0 0;
-    padding: 0;
+/*
+ * Header
+ */
+
+header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: .5rem;
 }
 
-#nav li {
-    font-size: 1.3em;
-    list-style: none;
-    float: left;
-    margin-top: 1.8em;
-    margin-left: 5px;
-    border-right: 2px solid #3A8ECD;
+.logo {
+    max-width: 25rem;
 }
 
-#nav li:last-child {
-    border-right: 0;
+/*
+ * Nav
+ */
+
+nav ul {
+    display: flex;
 }
 
-#nav li a {
-    font-size: 1.2em;
-    margin: 0.4em 0.4em;
-    display: block;
+nav ul li {
+    list-style: none;
+}
+
+nav ul li a {
     color: #FFF;
+    padding: .4rem .8rem;
     text-decoration: none;
     border-bottom: 2px solid #2e2e2e;
+    transition: ease 150ms;
 }
 
-#nav li a:hover {
-    border-bottom: 2px solid #fff;
+nav ul li a:hover {
+    border-bottom: 2px solid #187b55;
```

Comment:
Let’s stick with the white border-bottom here, to not introduce too many different colors

Reply:
Reverted in ecfc2b99468f4082082a68c67ad5581804715415